src/daemon.c

```diff
@@ -156,9 +161,9 @@ help(int r) {
 
 static void
 parse_arg(int argc, char **argv, hsk_options_t *opt) {
-  const static char *optstring = "c:n:r:i:u:p:k:s:l:h"
+  const static char *optstring = "c:n:r:i:u:p:k:s:l:h:a"
 #ifndef _WIN32
-    "d"
+    ":d"
```

Comment:
this `:` is a bug fix, unless you ever tried using `-h` before you wouldn't know it was broken.